js/index.js

```diff
@@ -0,0 +1,193 @@
+setTimeout(() => {
+   document.querySelector('header').style.backgroundImage = `url('resources/background.png')`;
+}, 1000)
+
+//Map information
+
+const canvas = document.getElementById('map');
+const map = new harp.MapView({
+   canvas,
+   theme: "resources/theme.json",
+   maxVisibleDataSourceTiles: 40, 
+   tileCacheSize: 100
+});
+
+map.resize(window.innerWidth, window.innerHeight);
+window.onresize = () => map.resize(window.innerWidth, window.innerHeight);
+
+const omvDataSource = new harp.OmvDataSource({
+   baseUrl: "https://xyz.api.here.com/tiles/herebase.02",
+   apiFormat: harp.APIFormat.XYZOMV,
+   styleSetName: "tilezen",
+   authenticationCode: 'AfArIuzngG4gkdhlBZkysnc',
+});
+map.addDataSource(omvDataSource);
+
+
+const options = [
+   //Japan
+   { 
+      tilt: 10, 
+      distance: 4000000,
+      center: new harp.GeoCoordinates(37.141262, 137.587148),
+      angle: 160
+   },
+   //India
+   { 
+      tilt: 10, 
+      distance: 4000000,
+      center: new harp.GeoCoordinates(20.752106, 77.889426),
+      angle: 160
+   },
+   //Europe
+   { 
+      tilt: 10, 
+      distance: 5000000,
+      center: new harp.GeoCoordinates(48.684884, 9.811884),
+      angle: 160
+   },
+   //Bay Area
+   { 
+      tilt: 0, 
+      distance: 80000,
+      center: new harp.GeoCoordinates(37.791294, -122.358022),
+      angle: 160
+   }
+];
+
+/*
+
+
+*/
+
+const selected = options[Math.floor(Math.random() * options.length)];
+
+map.addEventListener(harp.MapViewEventNames.Render, () => map.lookAt(selected.center, selected.distance, selected.tilt, (selected.angle += 0.05)));
+map.beginAnimation();
+
+
+const projects = [
+   {
+      name: 'harp.gl',
+      description: '3D web map rendering engine',
+      language: 'TypeScript',
+      stargazers_count: 12,
+      forks_count: 5
+   },
+   {
+      name: 'TIN Terrain',
+      description: '3D web map rendering engine',
+      language: 'JavaScript',
+      stargazers_count: 12,
+      forks_count: 5
+   },
+   {
+      name: 'harp.gl',
+      description: '3D web map rendering engine',
+      language: 'TypeScript',
+      stargazers_count: 12,
+      forks_count: 5
+   },
+   {
+      name: 'OSS Scanner',
+      description: '3D web map rendering engine',
```

Comment:
description is wrong?

Reply:
I had this array of fake projects so I could work locally on my machine when I was on a plane. These are from previous commits. Is it not acceptable to have this in the history?

Reply:
Up to the repo owner I guess. You could otherwise simply fix that by ammending the commits and such. :)